server/src/main/java/eu/fasten/server/connectors/PostgresConnector.java

```diff
@@ -16,34 +16,32 @@
  * limitations under the License.
  */
 
-package eu.fasten.server.db;
+package eu.fasten.server.connectors;
 
+import java.sql.DriverManager;
+import java.sql.SQLException;
 import org.jooq.DSLContext;
 import org.jooq.SQLDialect;
 import org.jooq.impl.DSL;
 import org.postgresql.Driver;
 
-import java.sql.DriverManager;
-import java.sql.SQLException;
-
 public class PostgresConnector {
 
     /**
      * Establishes database connection.
      *
      * @param dbUrl URL of the database to connect
      * @param user  Database user name
-     * @param pass  Database user password
      * @return DSLContext for jOOQ to query the database
      * @throws SQLException             if failed to set up connection
      * @throws IllegalArgumentException if database URL has incorrect format and cannot be parsed
      */
-    public static DSLContext getDSLContext(String dbUrl, String user, String pass)
+    public static DSLContext getDSLContext(String dbUrl, String user)
             throws SQLException, IllegalArgumentException {
         if (!new Driver().acceptsURL(dbUrl)) {
             throw new IllegalArgumentException("Could not parse database URI: " + dbUrl);
         }
-        var connection = DriverManager.getConnection(dbUrl, user, pass);
+        var connection = DriverManager.getConnection(dbUrl, user, System.getenv("DBPASS"));
```

Comment:
`DBPASS` is very generic. Perhaps rename to `FASTEN_DBPASS`. It could also read `PGPASSWORD`, the Postgres equivalent.

Reply:
So it reads `FASTEN_DBPASS`, but if it cannot find it then `PGPASSWORD`. Something like that or only one of them?

Reply:
Exactly that!

Reply:
Implemented